stglib/vec/cdf2nc.py

```diff
@@ -20,38 +22,31 @@ def cdf_to_nc(cdf_filename, atmpres=False):
 
     ds = utils.create_nominal_instrument_depth(ds)
 
-    ds, T, T_orig = set_orientation(ds, ds["TransMatrix"].values)
-
-    u, v, w = aqdutils.coord_transform(
-        ds["VEL1"],
-        ds["VEL2"],
-        ds["VEL3"],
-        ds["Heading"].values,
-        ds["Pitch"].values,
-        ds["Roll"].values,
-        T,
-        T_orig,
-        ds.attrs["VECCoordinateSystem"],
-    )
+    ds = set_orientation(ds)
 
-    ds["U"] = xr.DataArray(u, dims=("time", "sample"))
-    ds["V"] = xr.DataArray(v, dims=("time", "sample"))
-    ds["W"] = xr.DataArray(w, dims=("time", "sample"))
+    ds = transform.coord_transform(ds)
 
     ds = aqdutils.magvar_correct(ds)
 
-    # Rename DataArrays for EPIC compliance
-    ds = aqdutils.ds_rename(ds)
+    ds = combine_vars(ds)
 
-    ds = dist_to_boundary(ds)
+    if ds.attrs["VECBurstInterval"] != "CONTINUOUS":
+        ds = dist_to_boundary(ds)
 
     ds = scale_analoginput(ds)
 
+    # Shape into burst if not continuous mode
+    if ds.attrs["VECBurstInterval"] != "CONTINUOUS":
+        ds = reshape(ds)
+
     # Drop unused variables
     ds = ds_drop(ds)
 
     ds = qaqc.drop_vars(ds)
 
+    # Rename DataArrays for EPIC compliance
+    ds = aqdutils.ds_rename(ds)
```

Comment:
Think about where this should be in the pipeline

Reply:
Needs to come after combine_vars, because it would rename COR1-3, AMP1-3, and VEL1-3 which we don't need since we are combining those variables. Needs to come after ds_drop, so COR1-3, AMP1-3, and VEL1-3 are removed and not renamed. 

Moved this up as far as I could without having to add to other defs.